integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.5.48 Version bump reason: [nrf] Bump west version
+0.5.49 Version bump reason: [Ameba] Revise 'otar' to 'ota_requestor'
```

Comment:
I thought ota_requestor  was a term used in chip SDK rather than eternal ambd sdk.

Do we  have some sort  of circular reference here (i.e. SDK points to chip somehow)?

Reply:
Yes, actually the change discussed in https://github.com/project-chip/connectedhomeip/pull/13408#discussion_r786764643 requires the change in ameba sdk. As the term 'matter_enable_ota_requestor' is enabled in ameba sdk.